src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Validation/ValidationStack.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.AspNetCore.Mvc.ModelBinding.Validation
 {
-    public class ValidationStack
+    internal class ValidationStack
```

Comment:
Did you confirm that @JeremySkinner is not using this?

Reply:
I'm not using it :)

Reply:
I verified it. The one type it's affected by is `RequiredValidatorAdapter` which I've filed a follow up issue for: #8744